data/xp-pen-artist12.tablet

```diff
@@ -0,0 +1,31 @@
+# XP-Pen
+# Artist 12
+#
+# sysinfo.BPdqfy3fcP
+# https://github.com/linuxwacom/wacom-hid-descriptors/issues/238
+
+[Device]
+Name=XP-Pen Artist 12
+ModelName=
+DeviceMatch=usb:28bd:080a:UGTABLET 11.6 inch PenDisplay Mouse;usb:28bd:080a:UGTABLET 11.6 inch PenDisplay;usb:28bd:080a:UGTABLET 11.6 inch PenDisplay;usb:28bd:080a:UGTABLET 11.6 inch PenDisplay Keyboard
+PairedIDs=
+Class=Bamboo
+Width=
+Height=
```

Comment:
Can you fill in width and height for these please? It's the marketing width (i.e. what it says on the cardboard box), in integers. And after that, please run `meson test -C builddir` to make sure all the other requirements are met, fixing these one-by-one through github is a bit tiring.

Reply:
I filled the width and height by rounding the values in the box (256 mm = 10.079... in -> 10 in and 144 mm = 5.669... in -> 6 in). I also ran `meson setup builddir` and `meson test -C builddir` with all tests giving "Ok".

Reply:
yep, thanks, 10x6 is good enough, no-one reads those values anyway... looks like the test suite is passing now, let's get this merged! thanks!